Gopkg.toml

```diff
@@ -12,7 +12,7 @@
 
 [[constraint]]
   name = "github.com/openfaas/faas"
-  version = "0.6.16"
+  version = "0.8.8"
```

Comment:
waiting for new `faas` release

Reply:
Thanks, it's now released and available on GitHub.